about.html

```diff
@@ -0,0 +1,213 @@
+                <h2>African Movies Festival 2023 Logo</h2>
+                <hr class="line">
+            </div>
+            <div class="about-logo-paragraph">
+                <p>
+                    African Movies Festival introduced a new logo -: an expression of the diversity in our people and
+                    film makers. We believe that Africa is one, and the only language that brings film makers together
+                    is "action." Our mission is to unify African movie makers and build a future of movie industry that
+                    embodies our culture, connection, and community.
+                </p>
+            </div>
+            <div class="about-logo-image">
+                <img src="./images/download.jpg" alt="">
+            </div>
+        </div>
+
+    </section>
+
+    <section class="stories">
+        <div class="stories-container">
+            <div class="story-title">
+                <h2>See our previous events and projects</h2>
+                <hr class="line">
+            </div>
+            <div class="story-paragraph">
+                <p>African Movies endeavors to make Africa the Film making center </p>
+            </div>
+
+
+            <div class="event-area">
+                <div class="event-pic">
+                    <div class="pic-1">
+                        <img src="./images/students.png" alt="image">
+                    </div>
+                    <div class="text">
+                        <h5>2022</h5>
+                        <p>Introduce movies and film in campus</p>
+                    </div>
+                </div>
+
+                <div class="event-pic">
+                    <div class="pic-1">
+                        <img src="./images/conference.png" alt="image">
+                    </div>
+                    <div class="text">
+                        <h5>2022</h5>
+                        <p>Previous Film event</p>
+                    </div>
+                </div>
+            </div>
+
+
+        </div>
+
+
+        <!-- desktop  story images container-->
+
+        </div>
+
+    </section>
+
+    <section class="partners container" id="footer">
+        <div class="partners-content">
+        <h2>Partners</h2>
+        <hr class="lineb">
+
+        <div class="partners-logos">
+          <img src="./images/mozilla.svg" alt="logo">
+          <img src="./images/google.svg" alt="logo">
+          <img src="./images/naver.svg" alt="logo">
+          <img src="./images/airbnb.svg" alt="logo">
+           <img src="./images/kakao.svg" alt="logo">
+        </div>
+          </div>
+      </section>
+
+    <footer class="footer">
+        <div class="footer-container">
+            <div class="footer-logo">
+                <img src="./images/cc-logo2.png" alt="">
+            </div>
+
+            <div class="copyyright">
+                <p>2023 African Movies. Some Rights Reserved.</p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] @kazaneza great work implementing the changes requested however you forgot to change the background to the dark color, 

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/212414895-5831392d-a2d1-45eb-af8b-231d8a5c8f0e.png) |  ![image](https://user-images.githubusercontent.com/61408860/212414978-2f7d0eb0-6f6e-4f1c-bdcf-a85be83e1449.png)